src/components/DrawableSVG/DrawableSVG.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect } from 'react';
+import { useScrollTrack } from '../../utilities/customHooks/useScrollTrack';
+import './DrawableSVG.scss'
+
+interface DrawableSVGProps {
+  type: string;
+};
+
+export default function DrawableSVG({type}:DrawableSVGProps) {
+  const scrollPos = useScrollTrack();
+  useEffect(() => {
```

Comment:
I don't think this is unmounting correctly and is causing a memory leak warning. 
Have set up as a separate bug ticket: https://aegis-dev.atlassian.net/browse/AEG-47

Reply:
Will update on new ticket.